code/src/components/HeartCounter.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const HeartCounter = ({ heartCount, setHeartCounter }) => {
+  const removeHeartCounter = () => {
+    setHeartCounter('hidden');
+  };
+
+  // After 5000ms - set the heart-counter to hidden
+  setTimeout(removeHeartCounter, 5000);
```

Comment:
Nice function that it disappears while moving down on the site. Smart way to solve it! 👍🏼